### ISSUE TITLE
[jsx-a11y/label-has-associated-control] Doesn't detect children nested inputs

### ISSUE THREAD

**Issue by SBoudrias** [evcohen/eslint-plugin-jsx-a11y#482]
This rule fails for such cases:

```jsx
<label>
	First name:
	<span>
		<input type="text" />
	</span>
</label>
```

But I do believe that would be valid HTMl no?

**Comment by ljharb** [comment]
What’s your rule config? Note if you’re using the airbnb config, inputs need to *both* be nested and be for-id linked.

**Comment by SBoudrias** [comment]
@ljharb you're right, it's working fine when I set the config manually without using the airbnb config. Thanks for pointing this out :)

**Comment by ljharb** [comment]
np. if you for-ID link the input and the label (which is also necessary to get 100% a11y coverage) then it will work with the airbnb config, too.
